Duplicati/GUI/Duplicati.GUI.TrayIcon/HttpServerConnection.cs

```diff
@@ -439,14 +438,8 @@ public string StatusWindowURL
         {
             get 
             { 
-                try

```

Comment:
Generally is not good to have empty catches, this is why I removed these try-catch and allow Exception to be bubble to RealMain, where WebException is caught.  There is this reSpawn < 3 - idea was to give some option to absorb eg network problems, but after 3 tries to fail. This will not happen now, because empty catch will suppress this fail.
Im thinking about add some code to inform user, or add logging to TrayIcon, because no info is passed to user.